README.md

```diff
@@ -133,7 +133,8 @@ ExternalDNS publishes DNS records for applications to DNS servers, using a decla
 
 ### External DNS Versions
```

Comment:
Not related to this change, but I just noticed we still have this title as "Service Discovery Service" in the table of contents (line 12). Could you adjust that to match this section?

Reply:
Good catch! Fixed the table of contents.